CMS/urls.py

```diff
@@ -35,10 +35,9 @@
     path('api/token/', MyTokenObtainPairView.as_view(), name='token_obtain_pair'),
     path('api/token/refresh/', TokenRefreshView.as_view(), name='token_refresh'),
 
-    path('blog/',include('blog.urls')),
+    path('api/', include('blog.urls')),
 
-
-    path("", include("accounts.urls")),
+    path("api/", include("accounts.urls")),
```

Comment:
mbona hivi?

Reply:
> mbona hivi?

Just creating a better appealing URL, like /api/users or /api/blog/

Reply:
What if there's like an empty url in both account.urls and blog.urls won't there be like a conflict in the urls?

Reply:
> What if there's like an empty url in both account.urls and blog.urls won't there be like a conflict in the urls?

So you also saw it..

Reply:
This is the small change I have made. 
```
  path('api/', include('blog.urls')),
  path("", include("accounts.urls")),
```
This is the outcome
![Screenshot from 2023-02-10 09-05-36](https://user-images.githubusercontent.com/15798975/218015651-8cdfb7f2-1adc-481c-9770-88c52bb53583.png)